CHANGELOG.rst

```diff
@@ -0,0 +1,13 @@
+CHANGELOG
+=========
+
+This document describes changes between each past release.
+
+1.0.0 (2015-04-13)
```

Comment:
Are you sure you want to do a 1.0? Maybe a first round of release could let it mature.